packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -157,6 +158,11 @@ class CupertinoApp extends StatefulWidget {
   /// {@macro flutter.widgets.widgetsApp.localizationsDelegates}
   final Iterable<LocalizationsDelegate<dynamic>> localizationsDelegates;
 
+  /// {@macro flutter.widgets.widgetsApp.localeListResolutionCallback}
+  ///
+  /// This callback is passed along to the [WidgetsApp] built by this widget.
+  final LocaleListResolutionCallback localeListResolutionCallback;
+
   /// {@macro flutter.widgets.widgetsApp.localeResolutionCallback}
   ///
   /// This callback is passed along to the [WidgetsApp] built by this widget.
```

Comment:
Maybe add a comment here that, if possible, localeListResolutionCallback should be used instead.

Reply:
This still seems to be missing?

Reply:
Good catch, forgot to pass the list callback from CupertinoApp and MaterialApp. Definitely would have been a bug later!